### PR TITLE
Turbo stETH Permission for RY BTC/ETH

### PR DESCRIPTION
**Pull request by cbrit**

### CODE REVIEW

File: steward/src/cellars.rs [hunk 1]
```diff
@@ -40,11 +40,11 @@ pub const ALLOWED_CACHE_PRICE_ROUTER: [&str; 1] = [CELLAR_RYETH];
 // permissions
 
 pub const ALLOWED_V2_0_SETUP_ADAPTORS: [(&str, &str); 1] = [(CELLAR_RYUSD, ADAPTOR_CELLAR_V2)];
-pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 0] = [];
-pub const ALLOWED_V2_5_CATALOGUE_ADAPTORS: [(&str, &str); 2] = [
-    (CELLAR_TURBO_GHO, ADAPTOR_VESTING_SIMPLE_V2),
-    (CELLAR_TURBO_SWETH, ADAPTOR_VESTING_SIMPLE_V2),
+pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 2] = [
```
Comment (EricBolten) [inline steward/src/cellars.rs]: Why is the length of this vec 2? Feels like that shouldn't even compile.

File: steward/src/cellars.rs [hunk 2]
```diff
@@ -40,11 +40,11 @@ pub const ALLOWED_CACHE_PRICE_ROUTER: [&str; 1] = [CELLAR_RYETH];
 // permissions
 
 pub const ALLOWED_V2_0_SETUP_ADAPTORS: [(&str, &str); 1] = [(CELLAR_RYUSD, ADAPTOR_CELLAR_V2)];
-pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 0] = [];
-pub const ALLOWED_V2_5_CATALOGUE_ADAPTORS: [(&str, &str); 2] = [
-    (CELLAR_TURBO_GHO, ADAPTOR_VESTING_SIMPLE_V2),
```
Comment (EricBolten) [inline steward/src/cellars.rs:45]: We should confirm the safe removal of the two adaptors here.

File: steward/src/cellars.rs [hunk 3]
```diff
@@ -70,11 +70,12 @@ pub const ALLOWED_V2_0_POSITIONS: [(&str, u32); 20] = [
     (CELLAR_RYUSD, 28),
     (CELLAR_RYUSD, 29),
 ];
-pub const ALLOWED_V2_2_CATALOGUE_POSITIONS: [(&str, u32); 0] = [];
-pub const ALLOWED_V2_5_CATALOGUE_POSITIONS: [(&str, u32); 2] = [
```
Comment (EricBolten) [inline steward/src/cellars.rs:74]: Same here re: confirming safe removal.

File: steward/src/cellars.rs [hunk 4]
```diff
@@ -120,6 +121,7 @@ pub const CELLAR_RYUSD: &str = "97e6e0a40a3d02f12d1cec30ebfbae04e37c119e";
 pub const CELLAR_RYBTC: &str = "0274a704a6d9129f90a62ddc6f6024b33ecdad36";
 pub const CELLAR_TURBO_SWETH: &str = "d33dad974b938744dac81fe00ac67cb5aa13958e";
 pub const CELLAR_TURBO_GHO: &str = "0c190ded9be5f512bd72827bdad4003e9cc7975c";
+pub const CELLAR_TURBO_STETH: &str = "fd6db5011b171b05e1ea3b92f9eacaeeb055e971";
```
Comment (EricBolten) [inline steward/src/cellars.rs:124]: I don't see this used anywhere, is this just for reference?

**Review by EricBolten — COMMENTED** [review]
Some comments and questions.

File: steward/src/cellars.rs (hunk 4 above)
Comment (cbrit) [reply steward/src/cellars.rs:124]: yreah

File: steward/src/cellars.rs (hunk 1 above)
Comment (cbrit) [reply steward/src/cellars.rs]: fixed

**Review by josephterrigno — APPROVED** [review]

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**